azext_iot/operations/hub.py

```diff
@@ -2854,13 +2856,14 @@ def _iot_hub_distributed_tracing_show(target, device_id):
 
 
 def _validate_device_tracing(target, device_twin):
-    if target["location"].lower() not in TRACING_ALLOWED_FOR_LOCATION:
+    # Question: we can either force the hub to be found or skip checks
```

Comment:
Looks like RP info is needed for this validate, only hostname path isn't going to be optimal, so I lean towards force_find_resource=True.

Reply:
I will follow how I deal with missing idscope in dps device registration if the user gives the connection string. There are few examples of extra variables